### PR TITLE
Core/Scripts: Change door ID and name in Blackwing Lair

### PR DESCRIPTION
**Pull request by Exodius**
as per 7d454c66f70c19c8ff402fd126392e27700398ba

### CODE REVIEW

File: src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/blackwing_lair.h [hunk 1]
```diff
@@ -59,7 +59,7 @@ enum CreatureIds
 enum GameObjectIds
 {
     GO_BLACK_DRAGON_EGG         = 177807,
-    GO_BOSSGATE01               = 175946,
+    GO_BOSSGATE01               = 176965,
```
Comment (joschiwald) [inline src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/blackwing_lair.h]: pls update the name too

Comment (Exodius) [inline src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/blackwing_lair.h]: What is the correct name? Don't have the DB installed and, to be honest, I don't have enough drive space to re-install everything...

Comment (MitchesD) [inline src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/blackwing_lair.h]: @Exodius http://i.imgur.com/kiT81g7.png

Comment (tkrokli) [inline src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/blackwing_lair.h]: OK, so maybe a name like `GO_PORTCULLIS` would be useful here.

**Comment by Exodius** [comment]
@joschiwald Edited and squashed. Is it OK like this?

**Comment by Exodius** [comment]
Or should it be a bit flexible and be named GO_RAZORGORE_PORTCULLIS?

**Comment by tkrokli** [comment]
I think the name `GO_PORTCULLIS` is more flexible and that `GO_RAZORGORE_PORTCULLIS` is more specific.
But that is just my opinion. :)

**Pull request merged**